scripts/ci/finish-release.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/env/var bash
```

Comment:
Shall we start writing new scripts are javascript or semantic release plugins?

Reply:
would love to. this was only copy&paste 🚼